README.md

```diff
@@ -83,7 +83,7 @@ Using Cobra is easy. First, use `go get` to install the latest version
 of the library.
 
 ```
-go get -u github.com/spf13/cobra@latest
+go install github.com/spf13/cobra@latest
```

Comment:
`go install` is used to install executables: since `spf13/cobra` is a Go library for command line APIs, this is used to add the library to a go mod/sum:

> In Go 1.18, go get will no longer build packages; it will only be used to add, update, or remove dependencies in go.mod

Closing.

Reply:
Thanks for the explanation🙂

Reply:
Anytime!